### DISCUSSION TITLE
如果被看到这条讨论，那么，请注意下

### DISCUSSION THREAD

**Discussion by ZhangHappyHi** [Meloong-Git/PCL#2709] [讨论]
我发现下载材质包很麻烦，所以请求把材质包下载加入到【下载】的【资源】一栏中，和【mod】【整合包】在一起。
我知道这难为龙猫了，所以我只是提一下，如果有人认为这对游戏体验没有帮助的话就算了吧

**Comment by WForst-Breeze** [comment]
可以去新功能投票部分投票

**Reply by MoYuan-CN** [reply]
投票帖快链：#1095
